src/main/java/uk/gov/hmcts/reform/blobrouter/config/BlobDestination.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.blobrouter.config;
+
+public enum BlobDestination {
+    BULKSCAN,
```

Comment:
do you meant CFT here instead of bulkscan ?

Reply:
I originally called it CFT, but then realised the terminology we've been using in this repository is bulkscan. Don't mind changing it, but we'll lose some consistency.

Reply:
worth adding a javadoc comment on this class imho.

Reply:
Will do. Maybe we can decide what convention we want to go for here?

Reply:
Plus, any ideas for name to replace `blob-destination` with?

Reply:
perhaps `target-storage-account`?

Reply:
and value `bulkscan` (for the storage account) is clearer to me